andela/ah-backend-realers
# #166187853 signup validations

What does this PR do?
- Users should receive a descriptive error message when they enter invalid user parameters in an attempt to signup.

#### Description of Task to be completed?
- User should get descriptive validation errors messages on registration when they enter invalid user parameters when signing up.

#### How should this be manually tested?
1. Run `git fetch `
2. Then check out  `ft-signup-validations-166187853`
 Then using postman run this `http://127.0.0.1:8000/api/users/ `providing wrong data

#### What are the relevant pivotal tracker stories?
[# Finishes 66187853]

#### Screenshots (if appropriate)
<img width="957" alt="Screenshot" src="https://user-images.githubusercontent.com/43774983/59163801-56963b80-8b0e-11e9-9356-6f359857fac0.png">

#### Labels
Work in progress
